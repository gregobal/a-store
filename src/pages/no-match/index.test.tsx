import {render} from "@testing-library/react";
import {MemoryRouter} from "react-router-dom";
import {NoMatchPage} from "./index";

describe("NoMatch Page", () => {
    it("snapshot", () => {
        const view = render(
            <MemoryRouter>
                <NoMatchPage />
            </MemoryRouter>
        );

        expect(view).toMatchSnapshot();
    });
});
