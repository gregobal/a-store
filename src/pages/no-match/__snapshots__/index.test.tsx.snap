// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoMatch Page snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <h1
        class="typography__component_h1nzy typography__styrene-xlarge_qohs3 typography__bold_h1nzy"
      >
        Страница не найдена
      </h1>
      <span
        class="typography__primary-medium_xcdcz typography__bold_xcdcz"
      >
        Мы еще не сделали такую страницу, что бы снова не потеряться - воспользуйся Меню или начни с 
        <a
          class="link__component_1r33z link__primary_1b8mz"
          href="/"
        >
          <span
            class="link__text_1r33z"
          >
            Главной страницы
          </span>
        </a>
      </span>
    </div>
  </body>,
  "container": <div>
    <h1
      class="typography__component_h1nzy typography__styrene-xlarge_qohs3 typography__bold_h1nzy"
    >
      Страница не найдена
    </h1>
    <span
      class="typography__primary-medium_xcdcz typography__bold_xcdcz"
    >
      Мы еще не сделали такую страницу, что бы снова не потеряться - воспользуйся Меню или начни с 
      <a
        class="link__component_1r33z link__primary_1b8mz"
        href="/"
      >
        <span
          class="link__text_1r33z"
        >
          Главной страницы
        </span>
      </a>
    </span>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
